.github/workflows/go.yml

```diff
@@ -22,6 +22,14 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
+    - name: Ensure that all files are properly formatted
+      run: |
+        FILES=$(gofmt -s -l .)
+        if [ "$(gofmt -s -l . | wc -l)" -gt 0 ]; then
```

Comment:
This will run gofmt twice, right? Can we do `[ ! -z "${FILES}" ];` maybe?

Reply:
yeah that is definitely better.